### PR TITLE
MULTIARCH-3701: Enable ppc64le for agent installer

### PR DESCRIPTION
**Pull request by cs-zhang**
Enable openshift-install to be able to create agent based install ISO for Power, and it only supports platform none.

### CODE REVIEW

**Comment by cs-zhang** [comment]
Don't know about it for now.

File: pkg/asset/agent/installconfig.go [hunk 1]
```diff
@@ -95,6 +95,11 @@ func (a *OptionalInstallConfig) validateSupportedPlatforms(installConfig *types.
 	if installConfig.Platform.Name() != "" && !IsSupportedPlatform(HivePlatformType(installConfig.Platform)) {
 		allErrs = append(allErrs, field.NotSupported(fieldPath, installConfig.Platform.Name(), SupportedInstallerPlatforms()))
 	}
+
+	if installConfig.Platform.Name() != none.Name && installConfig.ControlPlane.Architecture == types.ArchitecturePPC64LE {
+		allErrs = append(allErrs, field.Invalid(fieldPath, installConfig.Platform.Name(), fmt.Sprintf("CPU architecture \"%s\" only supports platform \"%s\".", types.ArchitecturePPC64LE, none.Name)))
+	}
+
```
Comment (pawanpinjarkar) [inline pkg/asset/agent/installconfig.go:102]: `validateSupportedArchs()` is a better place to have this check.

**Pull request merged**